contracts/zero-ex/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "0.21.0-bsc.0",
```

Comment:
To publish the `bsc` branch:
- Go to GH actions
- Click `publish` workflow
- Enter `branch: bsc`, `prerelease: bsc`
- hit run and wait :crossed_fingers: 
- published packages will have versions `x.x.x-bsc.x`

Reply:
Note that we jump ahead of non-prerelease changelog entry even if that one's not published since these changes won't get included in the main branch publish.

Reply:
We technically don't need to add the `-bsc.0` suffx here in the changelog. Even this was just `x.x.x`, the publish script will still publish as `x.x.x-bsc.x`. But I recommend adding it here so when we eventually merge or rebase this branch with development we can tell the two changelog histories apart and don't accidentally merge two unrelated blocks.